src/main/java/br/com/caelum/comidinhas/tipoCozinha/NovoTipoCozinhaValidator.java

```diff
@@ -0,0 +1,25 @@
+package br.com.caelum.comidinhas.tipoCozinha;
+
+import org.springframework.validation.*;
+
+class NovoTipoCozinhaValidator implements Validator {
+
+    TipoCozinhaRepository tipoCozinhaRepository;
```

Comment:
Dependência como private e final.

```suggestion
    private final TipoCozinhaRepository tipoCozinhaRepository;
```